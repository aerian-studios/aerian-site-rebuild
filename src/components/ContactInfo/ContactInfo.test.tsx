import * as React from "react";

import * as renderer from "react-test-renderer";

import { ContactInfo } from "./index";

describe("ContactInfo", () =>
    it("renders correctly", () => {
        const tree = renderer
            .create(
                <ContactInfo
                    className="myClass"
                    title="Test title"
                    organisationName="test"
                    street="test"
                    locality="test"
                    postcode="test"
                    region="test"
                    phoneTitle="test"
                    phoneNumber="test"
                    emailTitle="test"
                    email="test"
                />
            )
            .toJSON();
        expect(tree).toMatchSnapshot();
    }));
