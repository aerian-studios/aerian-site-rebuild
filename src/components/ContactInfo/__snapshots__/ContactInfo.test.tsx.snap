// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactInfo renders correctly 1`] = `
<div
  className=" myClass"
>
  <h2>
    Test title
  </h2>
</div>
`;
